reframe/core/buildsystems.py

```diff
@@ -93,9 +175,55 @@ def _ldflags(self, environ):
 
 
 class Make(BuildSystem):
+    """A build system for compiling codes using ``make``.
+
+    The generated build command has the following form:
+
+    .. code::
+
+      make [-f MAKEFILE] [-C SRCDIR] [-j N] CC='X' CXX='X' FC='X' NVCC='X' CPPFLAGS='X' CFLAGS='X' CXXFLAGS='X' FFLAGS='X' LDFLAGS='X' OPTIONS
```

Comment:
The `-j` must be placed outside the square brackets since it is always generated.